cub/cub/device/dispatch/tuning/tuning_select_if.cuh

```diff
@@ -511,6 +518,232 @@ struct sm90_tuning<__uint128_t, flagged::yes, keep_rejects::yes, offset_size::_4
 {};
 #endif
 
+template <class InputT, flagged, keep_rejects, offset_size OffsetSize, primitive, input_size InputSize, may_alias>
+struct sm100_tuning;
+
+// select::if
+template <class Input>
+struct sm100_tuning<Input, flagged::no, keep_rejects::no, offset_size::_4, primitive::yes, input_size::_1, may_alias::no>
+{
+  // trp_0.ld_0.ipt_22.tpb_384.ns_0.dcid_2.l2w_915 1.099232  0.980183  1.096778  1.545455
+  static constexpr int threads                       = 384;
+  static constexpr int items                         = 22;
+  static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_DIRECT;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::exponential_backoff_constructor_t<0, 915>;
+};
+
+template <class Input>
+struct sm100_tuning<Input, flagged::no, keep_rejects::no, offset_size::_4, primitive::yes, input_size::_1, may_alias::yes>
+{
+  // trp_1.ld_0.ipt_20.tpb_448.ns_596.dcid_6.l2w_295  1.214635  1.001421  1.207023  1.307692
+  static constexpr int threads                       = 448;
+  static constexpr int items                         = 20;
+  static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_WARP_TRANSPOSE;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::exponential_backon_jitter_constructor_t<596, 295>;
+};
+
+template <class Input>
+struct sm100_tuning<Input, flagged::no, keep_rejects::no, offset_size::_4, primitive::yes, input_size::_2, may_alias::no>
+{
+  // trp_1.ld_0.ipt_20.tpb_256.ns_516.dcid_7.l2w_685 1.065598  0.937984  1.067343  1.452153
+  static constexpr int threads                       = 256;
+  static constexpr int items                         = 20;
+  static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_WARP_TRANSPOSE;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::exponential_backon_constructor_t<516, 685>;
+};
+
+template <class Input>
+struct sm100_tuning<Input, flagged::no, keep_rejects::no, offset_size::_4, primitive::yes, input_size::_2, may_alias::yes>
+{
+  // trp_1.ld_0.ipt_20.tpb_384.ns_1060.dcid_5.l2w_375 1.109871  0.973142  1.105415  1.459135
+  static constexpr int threads                       = 384;
+  static constexpr int items                         = 20;
+  static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_WARP_TRANSPOSE;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor = detail::exponential_backon_jitter_window_constructor_t<1060, 375>;
+};
+
+template <class Input>
+struct sm100_tuning<Input, flagged::no, keep_rejects::no, offset_size::_4, primitive::yes, input_size::_4, may_alias::no>
+{
+  // trp_1.ld_0.ipt_15.tpb_384.ns_1508.dcid_5.l2w_585 1.201993  0.920103  1.185134  1.441805
```

Comment:
During the verification process I run locally I will be posting results that I consider "better" and might fix unfortunate regressions. Will apply them after I finish screening. Same for subsequent comments.

// 1332  trp_1.ld_0.ipt_15.tpb_384.ns_2004.dcid_6.l2w_590 (400/1.db)  1.182044  0.987952  1.166869  1.433962